### ISSUE TITLE
Traffic path node delete bug

### ISSUE THREAD

**Issue by wyvernone** [dexyfex/CodeWalker#35]
If more than 2 traffic path nodes are deleted along with their path links, when saved and imported into the DLC rpf with openiv, the next time the project is loaded along with the DLC, there will be corrupted links pointing to those deleted nodes from other nodes.

**Comment by dexyfex** [comment]
Links are supposed to get deleted along with the nodes that they reference, will have to investigate further. Could it be links in other YND's are getting corrupted? Because that's a known issue since node id's all change when you delete a node, so other YND's could be referencing incorrect nodes after that.

**Comment by wyvernone** [comment]
Yes that could be it. Particularly the ones at the edge of an area.  Learning from this, if a node I want to delete is not the last, I'll just detach the links and move the node somewhere safe and leave it unused :-)

Thanks for looking. You can probably delete this bug report.

--------------------------------------------
On Thu, 20/12/18, dexyfex <notifications@github.com> wrote:

 Subject: Re: [dexyfex/CodeWalker] Traffic path node delete bug (#35)
 To: "dexyfex/CodeWalker" <CodeWalker@noreply.github.com>
 Cc: "wyvernone" <vtdone@yahoo.com>, "Author" <author@noreply.github.com>
 Received: Thursday, 20 December, 2018, 1:09 PM
 
 Links are supposed to get deleted
 along with the nodes that they reference, will have to
 investigate further. Could it be links in other YND's
 are getting corrupted? Because that's a known issue
 since node id's all change when you delete a node, so
 other YND's could be referencing incorrect nodes after
 that.
 
 —
 You are receiving this because you authored the thread.
 Reply to this email directly, view
 it on GitHub, or mute
 the thread.

**Comment by dexyfex** [comment]
Ideally we'd have it rebuilding links in all adjacent YND's whenever a node is deleted. Deleting nodes in existing YND's is currently quite problematic due to this issue... I guess I'll leave the issue here to remind me about it.
